common/src/main/java/nearsoft/skytouch/common/config/RabbitMQConfiguration.java

```diff
@@ -0,0 +1,8 @@
+package nearsoft.skytouch.common.config;
+
+public class RabbitMQConfiguration {
```

Comment:
Can these properties be put in the yml config file?

Reply:
Ok in the common's application.yml file right?